tests/testsuite.at

```diff
@@ -208,7 +208,9 @@ dnl Defines all possible Souffle flag configurations for testing.
 dnl NOTE: This is the default configuration that can be overridden
 dnl using SOUFFLE_CONFS environment variable.
 m4_define([DEFAULT_CONFS], [[], dnl interpreter
+ [-t],
```

Comment:
swap -t and -c for consistency.
you can comment in this file using dnl
[-t],                  dnl provenance